cli/src/test/lock.test.ts

```diff
@@ -0,0 +1,50 @@
+import anyTest, { TestInterface } from 'ava';
+import * as td from 'testdouble';
+import { Context, addFake } from './fake_lycan';
+import { createApp } from './utils';
+import { success } from 'specshell';
+import { UnauthorizedError, NotFoundError } from '@binaris/spice-koa-server/interfaces';
+
+const test = anyTest as TestInterface<Context>;
+
+addFake(test);
+
+const app = createApp({});
+
+const anything = td.matchers.anything();
+
+test.beforeEach(async (t) => {
+  t.assert(success(await t.context.shell.run(`cd ${t.context.projectDir}`)));
+  t.log(t.context.projectDir);
+});
+
+test('lockApp locks a unlocked application by given app name and lock reason', async (t) => {
```

Comment:
```suggestion
test('lockApp locks an unlocked application by given app name and lock reason', async (t) => {
```

Reply:
Can we have this test too?
`lockApp locks an unlocked application by current directory with given lock reason`